budget.json

```diff
@@ -4,7 +4,7 @@
         "timings": [
             {
                 "metric": "interactive",
-                "budget": 3000
+                "budget": 5000
```

Comment:
sometimes CI runs slow, but I think if my page takes more than 5 seconds to load I should know about it.